keras_cv/models/task.py

```diff
@@ -15,28 +15,18 @@
 
 import os
 
-from tensorflow import keras
-
+from keras_cv.backend import keras
 from keras_cv.utils.python_utils import classproperty
 from keras_cv.utils.python_utils import format_docstring
 
 
-@keras.utils.register_keras_serializable(package="keras_cv")
+@keras.saving.register_keras_serializable(package="keras_cv")
 class Task(keras.Model):
     """Base class for Task models."""
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self._backbone = None
-
-    @property
-    def backbone(self):
```

Comment:
We created this property for keras.io documentation (it only picks up properties). However, we might need to pick it up explicitly since I only see it on KerasNLP docs ([example](https://keras.io/api/keras_nlp/models/bert/bert_classifier/)).

Reply:
Ahh interesting -- I've reverted this and used an alternative workaround for the PyTorch insanity (setting `self._backbone` directly)

Reply:
If you want to keep the property, normal setters, and not annoy torch, this should work.

https://github.com/mattdangerw/keras-nlp/blob/b50f67d05027a8af1b59ba21e109183623cf07ef/keras_nlp/models/task.py#L88-L93

Reply:
Thanks for the point @mattdangerw!!